### PR TITLE
Rvinnakota/us111803 annotations can be added to an assignment

### PR DESCRIPTION
**Pull request by rohitvinnakota**
Old PR here:
https://github.com/BrightspaceHypermediaComponents/activities/pull/493

### CODE REVIEW

**Comment by rohitvinnakota** [comment]
Squashed old commits into a new one because CI was failing, thanks @ppaskaris-d2l

File: components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor-detail.js [hunk 1]
```diff
@@ -242,6 +252,17 @@ class AssignmentEditorDetail extends ErrorHandlingMixin(SaveStatusMixin(EntityMi
 					.token="${this.token}">
 				</d2l-activity-attachments-editor>
 			</div>
+
+			<div id="annotations-checkbox-container" ?hidden="${!this._canSeeAnnotations}">
+				<label class="d2l-label-text">${this.localize('annotationTools')}</label>
+					<d2l-input-checkbox
+						@change="${this._toggleAnnotationToolsAvailability}"
+						?checked="${this._annotationToolsAvailable}">
+						ariaLabel=${this.localize('annotationToolDescription')}>
```
Comment (ryantmer) [inline components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor-detail.js]: @rohitvinnakota This should be in quotes, I missed that 👀

Comment (ppaskaris-d2l) [reply components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor-detail.js]: hmm, doesn't inspire confidence when it works when it's wrong :/

Comment (rohitvinnakota) [reply components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor-detail.js]: Will get that fixed

**Pull request merged**